backend/cardflow/settings.py

```diff
@@ -81,7 +81,7 @@
         "NAME": os.getenv("POSTGRES_DB"),
         "USER": os.getenv("POSTGRES_USER"),
         "PASSWORD": os.getenv("POSTGRES_PASSWORD"),
-        "HOST": "postgres",
+        "HOST": os.getenv("POSTGRES_HOST"),
```

Comment:
Leave "postgres" because this targets the docker service name from the docker-compose, which will always be "postgres".

Reply:
Fixed.